runtime/doc/index.txt

```diff
@@ -1775,6 +1776,7 @@ tag		command		action ~
 |:winsize|	:wi[nsize]	get or set window size (obsolete)
 |:wincmd|	:winc[md]	execute a Window (CTRL-W) command
 |:winpos|	:winp[os]	get or set window position
+|:wlrestore|	:wl[restore]	restores the Wayland compositor connection
```

Comment:
Probably `restore` without `s`  should be used here as with all other descriptions?

Reply:
Is this expected to loose wayland compositor connectivity to justify a separate command to restore it?

Reply:
I guess it is ok if we have `:xrestore` command